sdk/cognitivelanguage/ai-language-conversations/review/ai-language-conversations.api.md

```diff
@@ -0,0 +1,1288 @@
+
+// @public (undocumented)
+export type AnalyzeConversationTaskUnion = AnalyzeConversationTask | ConversationalTask;
+
+// @public
+export interface AnswersOptions {
+    answerContext?: KnowledgeBaseAnswerContext;
+    confidenceThreshold?: number;
+    filters?: QueryFilters;
+    includeUnstructuredSources?: boolean;
+    qnaId?: number;
+    question?: string;
+    rankerKind?: RankerKind;
+    shortAnswerOptions?: ShortAnswerOptions;
+    top?: number;
+    userId?: string;
+}
+
+// @public
+export interface AnswerSpan {
+    confidence?: number;
+    length?: number;
+    offset?: number;
+    text?: string;
+}
+
+// @public
+export interface AnswersResult {
+    answers?: KnowledgeBaseAnswer[];
+}
+
+// @public
+export interface AreaResolution extends BaseResolution, QuantityResolution {
+    resolutionKind: "Area";
+    unit: AreaUnit;
+}
+
+// @public
+export type AreaUnit = string;
+
+// @public
+export interface AudioTiming {
+    duration?: number;
+    offset?: number;
+}
+
+export { AzureKeyCredential }
+
+// @public
+export interface BaseExtraInformation {
+    extraInformationKind: "EntitySubtype" | "ListKey";
+}
+
+// @public (undocumented)
+export type BaseExtraInformationUnion = BaseExtraInformation | EntitySubtype | ListKey;
+
+// @public
+export interface BasePrediction {
+    projectKind: "Conversation" | "Orchestration";
+    topIntent?: string;
+}
+
+// @public (undocumented)
+export type BasePredictionUnion = BasePrediction | ConversationPrediction | OrchestrationPrediction;
+
+// @public
+export interface BaseResolution {
+    resolutionKind: "Age" | "Volume" | "Speed" | "Area" | "Length" | "Information" | "Temperature" | "Weight" | "Currency" | "Boolean" | "DateTime" | "Number" | "Ordinal" | "TemporalSpan" | "NumericRange";
+}
+
+// @public (undocumented)
+export type BaseResolutionUnion = BaseResolution | AgeResolution | VolumeResolution | SpeedResolution | AreaResolution | LengthResolution | InformationResolution | TemperatureResolution | WeightResolution | CurrencyResolution | BooleanResolution | DateTimeResolution | NumberResolution | OrdinalResolution | TemporalSpanResolution | NumericRangeResolution;
+
+// @public
+export interface BooleanResolution extends BaseResolution {
+    resolutionKind: "Boolean";
+    // (undocumented)
+    value: boolean;
+}
+
+// @public
+export interface Conversation {
+    domain?: ConversationDomain;
+    id: string;
+    language: string;
+    modality: "text" | "transcript";
+}
+
+// @public
+export interface ConversationalTask extends AnalyzeConversationTask {
+    analysisInput: ConversationAnalysisOptions;
+    kind: "Conversation";
+    parameters: ConversationTaskParameters;
+}
+
+// @public
+export interface ConversationalTaskResult extends AnalyzeConversationTaskResult {
+    kind: "ConversationResult";
+    result: AnalyzeConversationResult;
+}
+
+// @public (undocumented)
+export class ConversationAnalysisClient {
```

Comment:
Could the client get documented? It is important to document everything in our public APIs to make it easier for our customers to use our APIs.

Reply:
Is there a standard procedure for documenting it?

Reply:
Perhaps a documentation similar to `TextAnalysisClient` in `@azure/ai-language-text` would be enough.